### PR TITLE
Added option to change spelling error severity

### PR DESCRIPTION
**Pull request by rmtsrc**
Show spelling errors with a blue underline (info severity), orange underline (warning severity), or red underline (error severity).

### CODE REVIEW

File: lib/providers.js [hunk 1]
```diff
@@ -318,7 +318,7 @@ export function provideLinter () {
               )
             }
             messages.push({
-              severity: 'warning',
+              severity: atom.config.get('linter-spell.showSpellErrorAsInfo') ? 'info' : 'warning',
```
Comment (yitzchak) [inline lib/providers.js]: Change to `severity: atom.config.get('linter-spell.spellingErrorSeverity'),`

File: package.json [hunk 2]
```diff
@@ -57,37 +57,44 @@
       "default": "",
       "order": 3
     },
+    "showSpellErrorAsInfo": {
+      "title": "Show spelling errors with blue underline (requires restart)",
+      "description": "Shows spelling errors with a blue underline (info severity), rather than the orange underline (warning severity).",
+      "type": "boolean",
```
Comment (yitzchak) [inline package.json]: Change to an enumeration with the values `info`, `warning` (default), `error` and call it `spellingErrorSeverity`. Update the documentation strings.

**Review by yitzchak — CHANGES_REQUESTED** [review]
Thanks for PR! I think we should probably just use an enumeration in case the user wants spelling errors to show as error in Linter. Please see my comments for details.

**Comment by rmtsrc** [comment]
@yitzchak Thanks for quick feedback! I've swapped the `boolean` for an `enumeration` with the default of `warning` like you suggested.

**Comment by yitzchak** [comment]
Awesome. Thanks again!

**Pull request merged**